modules/ppcp-api-client/src/Endpoint/class-requesttrait.php

```diff
@@ -36,6 +36,31 @@ private function request( string $url, array $args ) {
 			$args['headers']['PayPal-Partner-Attribution-Id'] = 'Woo_PPCP';
 		}
 
-		return wp_remote_get( $url, $args );
+		$response = wp_remote_get( $url, $args );
+
+		$this->logger->log( 'info', '--------------------------------------------------------------------' );
```

Comment:
Why log it as separate log entries? You can just add newlines. But both of these may not be a good idea because of the [DB logging mode](https://developer.woocommerce.com/2017/01/26/improved-logging-in-woocommerce-2-7/) (newlines are removed there iirc, and separate log entries will look weird + there are only like 10 entries per page). Maybe try putting it into more concise format without `----------` etc., just `GET URL ...., status code: ..., ...`. You can still use newlines so that it would look better at least in non-db mode, and in db view they should be displayed as spaces.

Reply:
Done.